src/nvim/main.c

```diff
@@ -186,6 +183,12 @@ int nvim_main(int argc, char **argv)
 int main(int argc, char **argv)
 #endif
 {
+  argv0 = basename(argv[0]);
```

Comment:
Given the null check, I should probably change the error message (assuming we don't just switch to path_tail).